app/config/security/accessControl.yml

```diff
@@ -2,18 +2,27 @@ security:
     access_control:
 
         # admin
-        login_area:
-            path: ^/%bamboo_admin_prefix%/login
+        admin_login_area:
+            path: ^/%bamboo_admin_prefix%/login.*
```

Comment:
Nopes, this is because maybe, in a future, we want to add some variations of login page... with this modification I can ensure I will be able without changing the security (BCBreak)